contracts/interfaces/IExceptions.sol

```diff
@@ -217,6 +214,12 @@ error TooManyApprovedBotsException();
 /// @notice Thrown when attempting to give a bot unexpected permissions
 error UnexpectedPermissionsException();
 
+/// @notice Thrown when a custom HF parameter lower than 10000 is passed into the full collateral check
```

Comment:
Using exact numbers in comments is a bad practice, as once such number change, it would be hard to fix all the relevant comments.  Consider changing `10000` with `PERCENTAGE_FACTOR` or other appropriate constant name.